README.md

```diff
@@ -40,6 +32,47 @@ mvn clean install
 
 This will build the artifacts after validating all checks including unit tests and place it in the deployer/target directory.
 
+
+## Installation
+
+* Build the project.
+* Login cf and select the space that Ocopea will deploy to.
+* Ensure a Postgres service and a MySQL service named 'crb-mysql' exist in the selected space. Run
+  ```
+  cf s
+  ```
+  Here is a sample output:
+  ```
+  name        service         plan                bound apps   last operation
+  crb-mysql   p-mysql         pre-existing-plan                create succeeded
+  postgres    user-provided
+  ```
+
+* From the deployer/target/classes folder, execute deploy.sh script:
+  ```
+  cd deployer/target/classes
+  ./deploy.sh
```

Comment:
Does this require bash shell to be present or can one run it from windows or mac? Just make sure you list the pre-req accordingly.

Reply:
Fixed the issue

Reply:
The docker image needs to have cf cli installed in order to run the script, so Ubuntu images won't work. Removed the note.